### ISSUE TITLE
jupypyter lab build fails with a linked local extension

### ISSUE THREAD

**Issue by halhenke** [jupyterlab/jupyterlab#4739]
Edit - Should have added the following info:

OS X 10.13.5
jupyterlab 0.32.1
installed via pip
using pyenv & python 3.6.5

---

Working on an extension based on the cookiecutter product. 

When `jupyter lab build` has to resolve with the extension it hangs indefinitely: 

```
[LabBuildApp] > node /Users/halhenke/.pyenv/versions/3.6.5/lib/python3.6/site-packages/jupyterlab/staging/yarn.js install
yarn install v1.5.1
(node:11345) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
info No lockfile found.
[1/4] 🔍  Resolving packages...
⢀ jupyterlab_sample_extension@file:../extensions/jupyterlab_sample_extension-0.1.0-c783287fd28f2c656162d6b015a8a5b1d16c9e1b.tgz
```

`jupyter labextension uninstall jupyterlab_sample_extension` and then running build again leads to the build completing as expected.

**Comment by blink1073** [comment]
Hi @halhenke, does this still happen with JupyterLab 0.34?

**Comment by halhenke** [comment]
Hi @blink1073 - I just retested it and no - it now builds properly 👍

**Comment by vidartf** [comment]
Likely resolved by #4543.

**Comment by jasongrout** [comment]
Closing as resolved. Thanks!
